src/modules/Dashboard/features/RespondentData/RespondentDataReview/Review/Review.utils.test.ts

```diff
@@ -0,0 +1,15 @@
+import { format } from 'date-fns';
+import { DateFormats } from 'shared/consts';
+import { getTimeResponseItem } from './Review.utils';
+
+describe('getTimeResponseItem', () => {
+  test.each`
+    answer                                   | expected                                                 | description
+    ${undefined}                             | ${undefined}                                             | ${'should return undefined if answer is not provided'}
+    ${{ value: { hours: 12, minutes: 30 } }} | ${format(new Date().setHours(12, 30), DateFormats.Time)} | ${'should return the formatted time when answer contains hours and minutes'}
```

Comment:
can you pls move some answer and expected values to variables to make test.each to be more readable?